CONTRIBUTING.md

```diff
@@ -92,31 +92,39 @@ change goes in the Changelog.
 
 Any _new_ functionality goes here. This may be a new field on a data type or a new data
 type altogether; a new API endpoint; or possibly a whole new feature. In general, these
-are sentence that start with the word "added." Examples:
+are sentence that start with the word "added."
+
+Examples:
 
 - `begin` field to silences that initiates silencing at a given timestamp
 - /healthz endpoint that reports health of the sensu-agent process
 
 ### Changed
 
 Changes to any existing component or functionality of the system that does not cause
-breaking changes to users or developers go here. _Changed_ is distinguishable from 
-_Fixed_ in that it is a change to intentional functionality. Examples:
+breaking changes to users or developers go here. _Changed_ is distinguishable from
+_Fixed_ in that it is a change to intentional functionality.
```

Comment:
existing instead of intentional? intentional change to existing functionality? just seems not quite right.